compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1211,3 +1211,12 @@ error.unsupported.primitive.type.reason=\
 
 error.method.signature.not.match=\
   '{ballerinax/java}'METHOD_SIGNATURE_DOES_NOT_MATCH ''{0}''
+
+warning.deprecated=\
+  ''{0}'' is deprecated
+
+error.deprecation.annotation.should.available=\
+  deprecation annotation should be available, since deprecation documentation is available
+
+error.deprecation.documentation.should.available=\
+  deprecation documentation should be available, since deprecation annotation is available
```

Comment:
``no `Deprecated` documentation found in deprecated construct``?

@MaryamZi @pubudu91 @rdhananjaya thoughts?

Reply:
```
invalid annotation: constructs annotated as '@deprecated' must have an accompanying 'Deprecated' documentation
```

Reply:
```
constructs annotated as '@deprecated' must have an accompanying 'Deprecated' documentation
```

Reply:
``constructs annotated as '@deprecated' must have 'Deprecated' documentation``

Reply:
"must have" or "must have a"

Reply:
documentation is non-countable? https://ell.stackexchange.com/questions/16580/difference-between-document-and-documentation

I think we should not have an "a".

Reply:
Changed